src/main/java/com/toptrumps/core/Player.java

```diff
@@ -1,4 +1,14 @@
 package com.toptrumps.core;
 
-public class Player {
-}
+
+import java.util.ArrayList;
+
+public class Player{
+
+    private ArrayList<Card> hand;
```

Comment:
Make this protected so it can be seen by AIPlayer. Add name instance variable and change constructor to suit

Reply:
Are we going to have a name for our Player class?

Reply:
I assume so? Both human and AI players will have a name right? If not ignore that comment and keep it how you’ve got it!

Reply:
Great catch @rjs68 ! Also, consider renaming it to "deck".

Reply:
For the deck class, is everyone cool if I just make it extend ArrayList?

Reply:
Yeah yeah, no need to have the "Deck" class